dojo/tools/trivy/parser.py

```diff
@@ -182,7 +182,7 @@ def get_result_items(self, test, results, service_name=None, artifact_name=""):
                             elif cvssclass.get("V2Score") is not None:
                                 severity = self.convert_cvss_score(cvssclass.get("V2Score"))
                             else:
-                                severity = self.convert_cvss_score(None)
+                                severity = TRIVY_SEVERITIES[vuln["Severity"]] # If not V2Score and V3Score assign Severity Default value to the severity variable
```

Comment:
Please take a look at this PR:
https://github.com/DefectDojo/django-DefectDojo/pull/9268